src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/help-maximumDeploymentSize.html

```diff
@@ -0,0 +1,6 @@
+<p>
+  The maximum number of instances of this template that can be deployed at one time. This can be used to avoid exceeding <a href="https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-best-practices#template-limits">Azure's Template Limits</a>.
```

Comment:
also very useful for quotas, what do you think?
```suggestion
  The maximum number of instances of this template that can be deployed at one time. This can be used to avoid exceeding <a href="https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-best-practices#template-limits">Azure's Template Limits</a> and resource quotas.
```

Reply:
Yes, this is what caused me to create this PR. The plugin was trying to deploy so many new VMs (nearly 300 in my case) in one deployment that it exceeded the 800 resources limit and Azure refused the deployment.

Reply:
I applied your suggestion, want me to resolve this conversation or do you want to?